docs/home/api-info/webhooks.mdx

```diff
@@ -57,10 +57,11 @@ to your desired URL making another request to the same endpoint.
 
 Webhook requests contain the relevant information on the body of the HTTP request.
 
-There are three types of messages you can expect to receive:
+There are four types of messages you can expect to receive:
 
 - [`ping`](#the-ping-message): validation of the webhook connection between Metriport and your app;
-- [user data](#devices-api-messages): health information grouped by user (Devices API only).
+- [user health data](#devices-api-messages): health information grouped by user (Devices API only).
+- [provider connected message](#devices-api-messages): notification of a newly-connected health provider by a user (Devices API only).
 - [patient data](#medical-api-messages): patient document information (Medical API only).
```

Comment:
since this list will only grow in the future, perhaps we can just link to the #devices-api-messages & #medical-api-messages sections below, and list the different messages in the sections?

seems like this would make it easier for a dev to understand

Reply:
nice